Table and Resolution Select Improvements

incognitojam opened the pull request:
I've edited the table so that the column titles are more descriptive, and hovering over the acronyms results in a small OS-independant pop-up.

I've also added some more common screen resolutions - I noticed a few were missing that I've used in the past.

incognitojam committed:
Table and Resolution Select Improvements

I've edited the table so that the column titles are more descriptive, and hovering over the acronyms results in a small OS-independant pop-up.

I've also added some more common screen resolutions - I noticed a few were missing that I've used in the past.

LeaVerou commented on index.html:
```diff
@@ -72,10 +74,10 @@
 					<th>Resolution</th>
-					<th>DPI</th>
-					<th>dppx</th>
+					<th><span title="Dots Per Inch">DPI</span></th>
```
Why not add the tooltip on the `th` directly? We don’t need a separate element here.

incognitojam replied:
I've made the changes accordingly. (Sorry, I'm a bit under-experience with HTML).

LeaVerou commented on index.html:
```diff
@@ -72,10 +74,10 @@
 				<tr>
 					<th>Name</th>
-					<th>Diagonal</th>
+					<th>Diagonal (inches)</th>
```
This will stretch the size of the column unnecessarily. Better as a tooltip.

incognitojam replied:
Thanks for the comment - I've made the changes.

LeaVerou commented:
Thanks! See inline comments.

incognitojam committed:
Removed unnecessary elements

I have moved the tooltips to the <th> element like suggested, and created a tooltip for the "(inches)" text on the diagonal column.

incognitojam commented:
I've now made all of the changes you suggested.

LeaVerou commented on index.html:
```diff
@@ -26,8 +26,10 @@
+		<a href="#" tabindex="-1">1680×900</a>
 		<a href="#" tabindex="-1">1440×900</a>
 		<a href="#" tabindex="-1">1366×768</a>
+		<a href="#" tabindex="-1">1360×768</a>
```
I would be opposed to adding new predefined resolutions that are so similar to existing examples. But even if I wasn't, it's always a good practice to send separate PRs for separate changes.

LeaVerou commented:
Thanks! See inline comment.

LeaVerou commented:
Since the resolutions haven't been removed to this PR, I'm afraid I can't merge it. :(
Closing for now, for housekeeping.
Please reopen or send a new PR for the HTML changes, those were very welcome.
